README.md

```
@@ -1,2 +1,121 @@
-# abyssal-tome
+# Abyssal Tome
+
+**Abyssal Tome (AT)** is a Python-powered application for Arkham Horror: The Card Game. It provides a comprehensive and easily searchable database of game rulings, FAQs, and clarifications, sourced from official documents, ArkhamDB, and community discussions.
+
+This project aims to:
+*   Consolidate rulings from diverse sources.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)